crates/crochet/Cargo.toml

```diff
@@ -7,7 +7,7 @@ edition = "2021"
 strip = "debuginfo"
 
 [lib]
-crate-type = ["cdylib"]
+crate-type = ["cdylib", "rlib"]
```

Comment:
`"rlib"` is required for us to be able to import modules from `crochet::` from the tests/ folder.